src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/mapper/AdresseHjelper.kt

```diff
@@ -17,10 +20,9 @@ object AdresseHjelper {
     }
 
     fun borPåSammeAdresse(barn: BarnMedIdent, bostedsadresserForelder: List<Bostedsadresse>): Boolean {
-        if (harDeltBosted(barn)) {
-            return false
+        if (harDeltBostedNå(barn)) {
```

Comment:
Denne metoden brukes for å vise vilkårsinformasjon, så å sjekke noe mot `nå`, når man går in på en eldre behandling blir ikke helt relevant , og kan vise annet enn det som var relevant for det datoet behandlingen ble behandlet

Reply:
Jepp - helt enig. 👍 

I første omgang er dette bare en refaktorering -> endret navn på harDeltBosted til harDeltBostedNå (fordi det var det den svarte på) 

Vi implementerte en versjon med dato som input, men valgte å slette det og vente til vi starter på vilkår. 

Det som er veldig forskjellig er at vi sier true ikke false, men mener det er riktig å si at barn bor hos bruker hvis barn har delt bosted. ("Nå" må fikses)